packages/sanity/src/structure/components/pane/Pane.tsx

```diff
@@ -81,18 +75,12 @@ export const Pane = forwardRef(function Pane(
   const expanded = expandedElement === rootElement
   const collapsed = layoutCollapsed ? false : pane?.collapsed || false
   const nextCollapsed = nextPane?.collapsed || false
-  const forwardedRef = useForwardedRef(ref)
   const flex = pane?.flex ?? flexProp
   const currentMinWidth = pane?.currentMinWidth ?? currentMinWidthProp
   const currentMaxWidth = pane?.currentMaxWidth ?? currentMaxWidthProp
 
-  const setRef = useCallback(
-    (refValue: HTMLDivElement | null) => {
-      setRootElement(refValue)
-      forwardedRef.current = refValue
-    },
-    [forwardedRef],
-  )
+  // Forward ref to parent
+  useImperativeHandle(ref, () => rootElement!, [rootElement])
```

Comment:
This is pedantic, but would it be more correct to type the imperative handle as `HTMLDivElement | null`, rather than using a non-null assertion? My understanding is that this asserts `rootElement` is never `null`, but in reality, it is.

```suggestion
  useImperativeHandle<HTMLDivElement | null, HTMLDivElement | null>(ref, () => rootElement, [
    rootElement,
  ])
```

Reply:
Yes, you're correct. The parent `ref: ForwardedRef<HTMLDivElement>` mandates it's always assigned, so I considered changing it as well so it's fully consistent. But it caused other side-effects to the type checks that felt out of scope. Do you want me to do it in a follow-up?

Reply:
I wondered about that, too, but `ForwardedRef` appears to always produce a nullable result (which is correct for consumers of this component).

IMO it's probably not worth spending more time on 😃.